build-system/tasks/bundle-size/index.js

```diff
@@ -115,7 +124,7 @@ async function storeBundleSize() {
   const commitHash = gitCommitHash();
   log('Storing bundle sizes for commit', cyan(shortSha(commitHash)) + '...');
   try {
-    const requestPost = util.promisify(require('request').post); // Lazy-required to speed up task loading.
+    const requestPost = await getRequestPost();
     const response = await requestPost({
```

Comment:
Nit: You could pass the params to `getRequestPost` (and rename to `requestPost`):

```js
async function requestPost(...args) {
  const {default: request} = await import('request');
  return util.promisify(request.post)(...args);
}

Reply:
Final version worked:

![image](https://user-images.githubusercontent.com/26553114/111846281-d56ff900-88dc-11eb-9d9d-fb8f6bd1aacf.png)